appservice/consumers/roomserver.go

```diff
@@ -123,8 +123,39 @@ func (s *OutputRoomEventConsumer) filterRoomserverEvents(
 	return nil
 }
 
+// appserviceHasMembershipInRoom returns a boolean depending on whether a given
+// appservice has membership at the time a given event was created.
+func (s *OutputRoomEventConsumer) appserviceHasMembershipForEvent(ctx context.Context, event *gomatrixserverlib.HeaderedEvent, appservice config.ApplicationService) bool {
```

Comment:
`appserviceHasMembershipForEvent` or `appserviceHasMembershipInRoom`? Comments disagree.

Reply:
Also what does "has membership" mean? Joined? Invited? Left? They are all memberships...

Reply:
Changed function to reflect reality.